packages/spanner/src/transaction.js

```diff
@@ -223,25 +228,26 @@ Transaction.prototype.begin = function(callback) {
  * });
  *
  * //-
- * // If the callback is omitted, we'll return a Promise.
+ * // To use promises, simply return a Promise.
  * //-
- * database.runTransaction()
- *   .then(function(data) {
- *     var transaction = data[0];
+ * database.runTransaction(function(err, transaction) {
+ *   if (err) {
+ *     // Error handling omitted.
```

Comment:
You are a better representative of our promise users, so if what I'm suggesting is more annoying to you than what we have now, then let's just call it case closed and keep what we have. If you think it's worth talking about more, please open us an issue or just wait to see if an annoyed user brings it up for us, thus validating the discussion.